jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/core/JDTServicesManager.java

```diff
@@ -165,6 +165,17 @@ public List<String> getExistingContextsFromClassPath(String uri, List<String> sn
                 classpath.add(null);
             });
         }
+
+        // FOR NOW, append package name and class name to the list in order for LS to
+        // resolve ${packagename} and ${classname} variables
+        String className = unit.getElementName();

```

Comment:
Do we need a null check here?

Reply:
getElementName from a valid `ICompilationUnit` should never be null, it's JavaElement, it should always have a name, it's very common API and used everywhere